Assets/HoloToolkit/Utilities/Scripts/AdaptiveQuality/AdaptiveViewport.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+using System;
+using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+	/// <summary>
+	/// Changes the VR viewport to correlate to the requested quality, trying to maintain a steady framerate by reducing the amount of pixels rendered.
+	/// Uses the AdaptiveQuality component to respond to quality change events.
+	/// At MaxQualityLevel, the viewport will be set to 1.0 and will linearly drop of to MinViewportSize at MinQualityLevel
+	/// </summary>
+
+	public class AdaptiveViewport : MonoBehaviour
+	{
+		[SerializeField]
+		private int MaxQualityLevel = 5;
```

Comment:
It is strange to me that `MaxQualityLevel` and `MinQualityLevel` are defined here in addition to in `AdaptiveQuality`. Can you expose them as public property getters from `AdaptiveQuality` and reference those properties from within this class instead of re-defining them?